File: Data Structures & Algorithms/two-integer-sum/submission-0.go
func twoSum(nums []int, target int) []int {
	for i := 0; i < len(nums); i++ {
		complent := target - nums[i]
		for j := i + 1; j < len(nums); j++ {
			if complent == nums[j] {
				return []int{i, j}
			}
		}
	}
	return []int{}
}

File: Data Structures & Algorithms/binary-search/submission-0.cpp
class Solution {
public:
    int search(vector<int>& nums, int target) {
        // brute force go through the array and return 
        // the index of target O(N) complexity and O(1) space

        for (int i = 0; i < nums.size(); ++i) {
            if (nums[i] == target) {
                return i;
            }
        }
        return -1;
    }
};
